up-streamer/src/ustreamer.rs

```diff
@@ -712,6 +712,18 @@ impl UListener for ForwardingListener {
             FORWARDING_LISTENER_FN_ON_RECEIVE_TAG,
             &msg
         );
+
+        if msg.attributes.payload_format.enum_value_or_default() == UPayloadFormat::UPAYLOAD_FORMAT_SHM {
```

Comment:
Rust convention is maximum line length of 100 characters: https://doc.rust-lang.org/nightly/style-guide/

```suggestion
        if msg.attributes.payload_format.enum_value_or_default()
            == UPayloadFormat::UPAYLOAD_FORMAT_SHM
        {
            debug!(
                "{}:{}:{} Received message with type UPAYLOAD_FORMAT_SHM, \
                which is not supported. A pointer to shared memory will not \
                be usable on another device. UAttributes: {:#?}",
                self.forwarding_id,
                FORWARDING_LISTENER_TAG,
                FORWARDING_LISTENER_FN_ON_RECEIVE_TAG,
                &msg.attributes
            );
            return;
        }
```

Reply:
Thanks for pointing it out. I just resolved that in my latest commit.